Assets/Scripts/DayNightCycle/TimeStamps.cs

```diff
@@ -0,0 +1,93 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public static class TimeStamps
+{
+    public delegate void UpdateTime(int value);
+    public static UpdateTime hourTime;
+    public static UpdateTime minuteTime;
+    public static UpdateTime secondTime;
+    public static UpdateTime dayTime;
+    public static float wholeDayTime;
+    private static bool timeRunning;
+
+    private static int currentHours;
+    private static int currentMinutes;
+    private static int currentDays;
+    private static int currentSeconds;
+    private static float decimalT;
+
+    public static float dayToMinute = 1; //15 minutes is one day
```

Comment:
one day passed in about 1 minute, which is accurate but your comment did confuse me lol.

Reply:
My bad once again, will update the comment/s to acknowledge all the features